packages/vue/jest.config.cjs

```diff
@@ -1,4 +1,19 @@
 module.exports = {
+  collectCoverage: true,
+  collectCoverageFrom: ['<rootDir>/src/**/*.(ts|vue)'],
+  // ignore coverage for index, version and style files
```

Comment:
This note doesn't really match what's below 😅

Reply:
😅 Forgot to update. Thanks for catching it!